tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -69,14 +69,14 @@ CREATE TABLE IF NOT EXISTS internal_keys (
     key_index INTEGER NOT NULL
 );
```

Comment:
Does `key_family` need to be renamed?

Reply:
I don't think so, AFAICT that is related to key derivation and not asset grouping.

Reply:
Yeah, the `key_familiy` is a term from how we derive our private keys. That name is the main reason for the whole rename of "family key" to "group key" because "key family" and "family key" were just too easy to mix up and would often complicate things. For example the "family key" has a "key family" that is used for derivation. And now it would just be the "group key" has a "key family" that is used for derivation.

Reply:
Makes sense! :+1: